enterprise/migrations/0059_add_code_management_portal_config.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-12-14 17:46
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('enterprise', '0058_auto_20181212_0145'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='enterprisecustomer',
+            name='enable_portal_code_management_screen',
+            field=models.BooleanField(default=False, help_text='Specifies whether to allow access to the code management screen in the admin portal.'),
+        ),
+        migrations.AddField(
+            model_name='historicalenterprisecustomer',
```

Comment:
Just for context and my general understanding, what is the `historialenterprisecustomer` model for?

Reply:
it was auto generated by makemigrations because of this line: https://github.com/edx/edx-enterprise/blob/41580978a5ce1e30896e08c70399fa1e8a55a7d3/enterprise/models.py#L176
we have it presumably to keep the history of changes on instances of this model, but i'm not sure if we really use it or care about it.